pkg/client/client.go

```diff
@@ -22,8 +29,19 @@ func NewConcourseClient(
 		Endpoint: oauth2.Endpoint{TokenURL: url + "/sky/issuer/token"},
 		Scopes:   []string{"email", "federated:id", "groups", "openid", "profile"},
 	}
+	cacerts, err := getCaCert(caFile)
+
+	if err != nil {
+		panic("Cannot load cacerts from file " + caFile)
```

Comment:
Please can we avoid `panic` here. Return the `err` and let it be processed correctly rather than crashing the process.

Reply:
done.

Reply:
done.